client/my-sites/comments/main.jsx

```diff
@@ -10,21 +10,33 @@ import { localize } from 'i18n-calypso';
 import Main from 'components/main';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 import DocumentHead from 'components/data/document-head';
+import CommentList from './comment-list';
+import CommentNavigation from './comment-navigation';
 
 export class CommentsManagement extends Component {
 
 	static propTypes = {
 		basePath: PropTypes.string,
+		siteSlug: PropTypes.string,
+		status: PropTypes.string,
 		translate: PropTypes.func,
 	};
 
 	render() {
-		const { translate, basePath } = this.props;
+		const {
+			basePath,
+			siteSlug,
+			status,
+			translate,
+		} = this.props;
 		return (
-			<Main className="comments">
+			<Main className="comments" wideLayout>
 				<PageViewTracker path={ basePath } title="Manage Comments" />
 				<DocumentHead title={ translate( 'Manage Comments' ) } />
-				<div>Hello World!</div>
+				<div className="comments__primary">
+					<CommentNavigation siteSlug={ siteSlug } status={ status } />
+					<CommentList />
```

Comment:
Sure, but if that's the case I'd like us to explore some paging options sooner rather than later, since folks will have a lot of comments to moderate on an active site.

Do we go with simple paging, or try our hand at another InfiniteList implementation? The latter is often not trivial to implement, though having static item heights will help a lot.

For example if we have 100 comments, do we additively add them to the list and render them all, or do we use some more complex logic and only render what's in view, say a person looking at comments 50-80? Also, if we go with something more complex, how do we handle batch actions?